source/getting-started.txt

```diff
@@ -18,69 +19,175 @@ Get Started with Relational Migrator
-      fields in MongoDB documents.
+Convert and Validate SQL Objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. step:: Customize your mapping rules
+{+rel-mig+} uses generative AI to :ref:`convert <rm-query-converter>` SQL queries, 
+views, and stored procedures into MongoDB-compatible code for C#, Java, or JavaScript.
 
-      Add or update :ref:`mapping rules <rm-mapping-rules>` to create 
-      the destination data model that best matches your 
-      application needs.
+You can test queries within {+rel-mig+} to validate converted SQL and 
+ensure that your queries are compatible with MongoDB.
 
-   .. step:: Create a migration job
+Migrate to MongoDB
+~~~~~~~~~~~~~~~~~~
 
-      :ref:`Migration jobs <rm-sync-jobs>` move the data from relational 
-      tables to MongoDB collections. Migration jobs have two types: 
+Use {+rel-mig+} to migrate data from :ref:`supported relational databases <supported-databases>` 
+to MongoDB Atlas or a self-managed MongoDB deployment. You can choose either a 
+snapshot migration, or continuous synchronization with Change Data Capture (CDC).
+For more information, see :ref:`<rm-use-cases>`.
+
+{+rel-mig+} integrates with Apache Kafka and Confluent Cloud for large scale 
+migrations. For more information, see :ref:`<kafka-intro>`.
+
+.. _rm-use-cases:
+
+Use Cases
+---------
+
+{+rel-mig+} supports both Snapshot and Continuous migration jobs.
+
+.. include:: /includes/fact-short-sync-job-desc.rst
+
+Some relational databases are only compatible with Snapshot migration jobs.
+For more information, see the following table:
+
+.. include:: /includes/table-supported-migration-types.rst
+
+.. _rm-migration-scenarios:
+
+Migration Scenarios
+-------------------
+
+You can use {+rel-mig+} to migrate one legacy application at a
+time to MongoDB. Depending on your application's needs, you can migrate
+either with or without downtime.
+
+Migrate During Scheduled Downtime
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application does not require constant uptime, such as in the
+case of an internal application, you can have the migration occur during
+scheduled downtime. Migrating during downtime is the easiest way to
+configure and complete the migration process.
+
+When you start the migration process, {+rel-mig+} takes a
+snapshot of the data on the legacy application and migrates the data
+from that point in time. During the migration process, your legacy
+application can remain online to support incoming reads, but cannot
+support incoming writes. The length of the migration process depends on
+several factors, such as the amount of data being migrated and your
+connection speed.
+
+After the migration completes, take your legacy application offline and
+launch your new application.
+
+
+Migrate Without Downtime
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your application must remain online to support reads and writes
+during the migration process, you can use Change Data Capture (CDC) to
```

Comment:
(terminology)
A quick google search suggests that change data capture isn't used as a proper noun, usually. :)